src/services/protobufService.ts

```diff
@@ -153,7 +162,8 @@ class ProtobufService {
               name: schema.name,
               error: protoError
             };
-          })
+          }).filter(schema => schema.name != 'org/infinispan/protostream/message-wrapping.proto' &&
+            schema.name != 'org/infinispan/query/remote/client/query.proto')
```

Comment:
I'm holding off merging now, as I think this filtering is not needed. The issue is with the Quarkus client. See the #Schemas topic on zulip for more context.

Reply:
It will probably be needed until we can get the change added to a released version of Quarkus, which will be a while I am guessing :(